src/routes.tsx

```diff
@@ -1,38 +1,16 @@
 import { Route, Routes } from 'react-router-dom';
 
-import Landing from './components/main/Landing';
-import Login from './components/auth/Login';
-import TodosList from './components/todos/TodosList';
-import TodoCreate from './components/todos/TodoCreate';
-import TodoEdit from './components/todos/TodoEdit';
-import TodoDelete from './components/todos/TodoDelete';
-import MainBar from './components/main/MainBar';
-import UserInfo from './components/main/UserInfo';
+import Auth from './pages/auth/Aurh';
+import TodosPage from './pages/todos/TodosPage';
+import UpdateTodo from './pages/update-todo/UpdateTodo';
+import UserInfo from './pages/user-info/UserInfo';
 
 const AppRoutes = () => {
    return (
      <Routes>
-       <Route path="/" element={<Landing />} />
-       <Route path="/auth" element={<Login />} />
-       <Route
-         path="/todos"
-         element={
-           <>
-             <MainBar />
-             <TodosList />
-             <TodoCreate />
-           </>
-         }
-       />
-       <Route
-         path="/todos/:id"
-         element={
-           <>
-             <TodoEdit />
-             <TodoDelete />
-           </>
-         }
-       />
+       <Route path="/" element={<TodosPage />} />
+       <Route path="/auth" element={<Auth />} />
+       <Route path="/:todoid/update" element={<UpdateTodo />} />
```

Comment:
How do you delete? on the update route?

Reply:
for best practice we can to do array and map Routes like that exmaple [ {path:"/", element: <TodosPage/> }]